src/lib/make-welcome-notification.ts

```diff
@@ -0,0 +1,61 @@
+import Cookies from 'js-cookie'
+import { toast, ToastColor } from 'components/toast'
+
+const WELCOME_COOKIE = 'dev-dot-first-welcome-notification'
+const INITIALIZATION_COOKIE = 'dev-dot-welcome-notification-initialized'
+const DISMISS_COOKIE = 'dev-dot-dismiss-welcome'
+const SESSION_COOKIE = 'dev-dot-welcome-session'
+
+function permanentlyDismiss() {
+	// Set dismiss cookie that expires after a year assuming we won't need these notifications in a year after GA
+	Cookies.set(DISMISS_COOKIE, true, {
+		expires: 365,
```

Comment:
💭 if we want this to be 1 year from GA, perhaps we pass a date object instead?

Reply:
Yeah sounds good! We can also add a check to see if the current date is later than the date we want to remove these on. You can see that [here](https://github.com/hashicorp/dev-portal/pull/995/commits/95791b152223a33dacd60f704d0c168caf0b3a93#diff-b043befe629def842d164cb756567813f1e0dcacc5ea16d005ae2a99d8b7188dR38). I am confirming now that we won't want these in ~a year and the other question you asked regarding release 👍